content/opt/run

```diff
@@ -219,5 +212,5 @@ echo "==================================================================="
 touch ${initfile}
 fi
 
-echo "Starting Supervisor.  You can safely CTRL-C and the container will continue to run with or without the -d (daemon) option"
+echo "Starting Supervisor."
```

Comment:
I've heard docker minor versions are often not compatible to each other. In fact I'm quite new to docker software and I have only used the 1.12 version yet. But I have to say, that this is sadly not true in 1.12.

```
[bebe:~/code/icinga2-docker] devtest ± docker run --rm -ti jordan/icinga2
[...]
Starting Supervisor.  You can safely CTRL-C and the container will continue to run with or without the -d (daemon) option
^C
[bebe:~/code/icinga2-docker] devtest ± docker ps
CONTAINER ID        IMAGE               COMMAND             CREATED             STATUS              PORTS               NAMES
[bebe:~/code/icinga2-docker] devtest ±
```